11/nanoserver-1809/Dockerfile

```diff
@@ -33,7 +33,8 @@ RUN $output = net users ; `
     if(-not ($output -match $env:user)) { `
         Write-Host 'user does not exist?' ; `
         net user $env:user /add /expire:never /passwordreq:no ; `
-        net localgroup Administrators /add $env:user `
+        net localgroup Administrators /add $env:user ; `
+        wmic useraccount WHERE Name=$env:user set PasswordExpires=false; `
```

Comment:
I don't believe wmic is available in nanoserver images. Did you find otherwise?

Reply:
it built locally and on ci.  unless powershell is ignoring errors.

```
2020-06-25T13:21:05.791Z] Status: Downloaded newer image for jenkins/agent:4.3-4-jdk11-nanoserver-1809

[2020-06-25T13:21:05.791Z]  ---> 329af297a698

[2020-06-25T13:21:05.791Z] Step 3/9 : ARG VERSION

[2020-06-25T13:21:06.077Z]  ---> Running in a4a9bc59c369

[2020-06-25T13:21:08.086Z] Removing intermediate container a4a9bc59c369

[2020-06-25T13:21:08.086Z]  ---> 98a89a4cf873

[2020-06-25T13:21:08.086Z] Step 4/9 : LABEL Description="This is a base image, which allows connecting Jenkins agents via JNLP protocols on Windows" Vendor="Jenkins Project" Version="$VERSION"

[2020-06-25T13:21:08.086Z]  ---> Running in c157ea2c62f3

[2020-06-25T13:21:10.740Z] Removing intermediate container c157ea2c62f3

[2020-06-25T13:21:10.740Z]  ---> 63a20082ee57

[2020-06-25T13:21:10.740Z] Step 5/9 : ARG user=jenkins

[2020-06-25T13:21:10.740Z]  ---> Running in 806d56bcfd3a

[2020-06-25T13:21:13.401Z] Removing intermediate container 806d56bcfd3a

[2020-06-25T13:21:13.401Z]  ---> 33e0ef19a79a

[2020-06-25T13:21:13.401Z] Step 6/9 : RUN $output = net users ;     if(-not ($output -match $env:user)) {         Write-Host 'user does not exist?' ;         net user $env:user /add /expire:never /passwordreq:no ;         net localgroup Administrators /add $env:user ;         wmic useraccount WHERE Name=$env:user set PasswordExpires=false;     }

[2020-06-25T13:21:13.401Z]  ---> Running in b9a823a56143

[2020-06-25T13:21:25.870Z] Removing intermediate container b9a823a56143

[2020-06-25T13:21:25.870Z]  ---> 3ebd42fed6db

[2020-06-25T13:21:
```

Reply:
Interesting, that was my initial try at fixing this issue, but when I tried to build on nanoserver it had an issue. Perhaps they added it to newer version of the base image.

Reply:
I did not verify in the resukting image the account was correctly configured, however the command did not fail and the same command running in Windows 10 did set the flag